examples/java/README.md

```diff
@@ -2,17 +2,21 @@
 
 Ensure that you have an instance of Valkey running on "localhost" on "6379". Otherwise, update glide.examples.ExamplesApp with a configuration that matches your server settings.
 
-To run the example:
+To run a standalone example:
 ```
 cd valkey-glide/examples/java
-./gradlew :run
+./gradlew :runStandlone
+```
+To run a cluster example:
+```
+cd valkey-glide/examples/java
+./gradlew :runCluster
 ```
 
 You should expect to see the output:
```

Comment:
for the cluster we also test replication info. I think you should remove this whole "You should expect to see the output:" section, as it also adds another place to maintain which we would probably miss when we'll do changes in the examples.